backend/fms_core/templates.py

```diff
@@ -639,6 +639,7 @@
   # prefill_info : [("Template Sheet Name", "Template Column Header", "Queryset Name", "Sample Model Attribute/Property", "Extractor Function"), ...]
   "prefill info": [
       ("SampleQC", "Sample Name", "name", "name", None),
+      ("SampleQC", "Sample Kind", "kind", "kind", None),
```

Comment:
You should not have to add Sample kind here if the specification works correctly. kind is not a sample attribute so this likely does not work. You can remove this line and validate that the prefill in the workflow still works.